app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,45 @@
  *= require foundation_and_overrides
 
  */
+
+$margin:10%;
+body {
+ margin: $margin;
+}
+.profile-block-list {
+  text-decoration: none;
+  list-style-type: none;
+}
+@mixin div-blocks($border-color) {
+  border: 3px solid $border-color;
+  padding: 3%;
+  border-radius: 40px;
+}
+@mixin div-hovers {
+  padding: 3%;
+  box-shadow: 5px 5px 5px #bbb;
```

Comment:
Color literals like `#bbb` should only be used in variable declarations; they should be referred to via variable everywhere else.